src/types/questionnaire/base.ts

```diff
@@ -0,0 +1,5 @@
+export type Period = {
+  id?: string;
```

Comment:
Does it have an ID 🤔 

Coudn't find one in FHIR spec https://build.fhir.org/datatypes.html#period